### PR TITLE
ReadMe

### PR DESCRIPTION
**Pull request by polarbub**
update the readme and fix polar stupid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,28 @@ Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vul
 Early Development and is **NOT** intended for use by the faint-hearted. Support for Blaze4D can be found in the #support
 Discord channel.
 
-Note: Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+## Things that don't work
+- Sodium, Iris or Canvas probably don't work yet, so don't bother trying. We're trying to add support for them ourselves
+- Fancy graphics
```
Comment (hYdos) [inline README.md:10]: Fabulous is the only one which doesnt work.

File: README.md [hunk 2]
```diff
@@ -5,17 +5,28 @@ Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vul
...
+- MacOS doesn't work
+- F2 screenshots freeze your game
+- AMD gpus
```
Comment (hYdos) [inline README.md:13]: Some AMD cpu's aswell

**Review by hYdos — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (polarbub) [reply README.md:13]: I thought you patched that as my amd cpu that used to be broken now works.

File: README.md (hunk 1 above)
Comment (polarbub) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -5,17 +5,29 @@ Blaze4D is a Fabric mod that changes Minecraft's rendering engine to use the Vul
...
 
 ## Community
 We have a [Discord server](https://discord.gg/H93wJePuWf) where you can track development progress, ask questions, or just hang out in.
 
+## Mod Dependencies
+[Fabric Language Kotlin](https://modrinth.com/mod/fabric-language-kotlin)
+
 ## Building
-1. Clone the repository (https://github.com/Blaze4D-MC/Blaze4D.git)
+1. Clone the repository (https://github.com/Blaze4D-MC/Blaze4D.git). Make sure to use `--recursive` to also clone [Rosella]{https://github.com/Blaze4D-MC/Rosella}.
```
Comment (hYdos) [inline README.md:24]: This addition is no longer needed because we use a maven repository

**Review by hYdos — CHANGES_REQUESTED** [review]
e

File: README.md (hunk 3 above)
Comment (polarbub) [reply README.md]: Okay, fixed in the latest commit

**Comment by polarbub** [comment]
I don't know how to mark requested changes as done. Can some one make the changes as done?

**Review by hYdos — APPROVED** [review]

**Pull request merged**